src/types/page.urls.ts

```diff
@@ -81,7 +81,7 @@ export const CHECK_YOUR_ANSWERS_PATH_END = "director-check-answers";
 export const STOP_PAGE_END = "/cannot-use";
 export const BASIC_STOP_PAGE = COMPANY_AUTH_PROTECTED_BASE + STOP_PAGE_END;
 export const BASIC_STOP_PAGE_PATH = OFFICER_FILING + BASIC_STOP_PAGE + `?${URL_QUERY_PARAM.PARAM_STOP_TYPE}={${URL_QUERY_PARAM.PARAM_STOP_TYPE}}`;
-export const APPID_STOP_PAGE = ACTIVE_BASE + CONTAINS_APPOINTMENT_ID + STOP_PAGE_END;
+export const APPID_STOP_PAGE = ACTIVE_BASE + CONTAINS_SUBMISSION_ID + CONTAINS_APPOINTMENT_ID + STOP_PAGE_END;
```

Comment:
The Jira states we do not want the appointment ID in any stop page URL. Under root cause action section

Reply:
Thanks, Jack. updated.